src/features/game/events/chop.ts

```diff
@@ -66,17 +67,18 @@ export function chop({
   action,
```

Comment:
It could be useful to mark `state` as a readonly object now to prevent accidental usage in the future.

For example:

```
type Options = {
  state: Readonly<GameState>;
  action: ChopAction;
  createdAt?: number;
};
```

Reply:
Top suggestion 🥇